tensorflow/python/kernel_tests/scatter_nd_ops_test.py

```diff
@@ -232,6 +240,14 @@ def testVariableRankAdd(self):
   def testVariableRankSub(self):
     self._VariableRankTests(_NumpySub, state_ops.scatter_nd_sub)
 
+  @test_util.run_deprecated_v1
+  def testVariableRankMin(self):
+    self._VariableRankTests(_NumpyMin, state_ops.scatter_nd_min)
+
+  @test_util.run_deprecated_v1
+  def testVariableRankMax(self):
+    self._VariableRankTests(_NumpyMax, state_ops.scatter_nd_max)
```

Comment:
Test cannot be run on complex types, those should be removed from the test first